src/lib/containers/UpsetPlot.tsx

```diff
@@ -37,6 +38,7 @@ const UpsetPlot: React.FunctionComponent<UpsetPlotProps> = ({
 }) => {
   const [isLoading, setIsLoading] = useState<boolean>()
   const [data, setData] = useState<UpsetPlotData>()
+  const [error, setError] = useState<string>()
```

Comment:
Show a "danger" bootstrap alert if any error occurs when we get the plot data and transform.